admin_guide/seccomp.adoc

```diff
@@ -0,0 +1,149 @@
+[[admin-guide-seccomp]]
+= Restricting Application Capabilities Using Seccomp
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+[IMPORTANT]
+====
+Seccomp is
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[currently
+in alpha support].
```

Comment:
@pweil-  What's the policy of RH? Does it mean technical preview?

> The feature may have significant bugs and is suitable for testing and prototyping.

It looks like not production ready feature.